src/services/supabase.ts

```diff
@@ -44,7 +35,8 @@ export const ERROR_MESSAGES = {
 export const tokenHasIssues = (errorMessage?: string) => {
     return (
         errorMessage &&
-        (errorMessage === ERROR_MESSAGES.jwtExpired ||
+        (errorMessage.startsWith('JWSError ') ||
```

Comment:
Yes... because otherwise you would not have left this comment and I would not have noticed this being added and huge issues could have happened in prod LOL

I add errors like this only for testing. We should never check for errors that are not stored in the list up above
![image](https://github.com/estuary/ui/assets/270078/4874ddd7-dcb0-4e25-af20-9720906e27b0)